README.md

```diff
@@ -1 +1,135 @@
-# Leaderboard-website
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b></b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 Leaderboard Website <a name="about-project"></a>
+
+In this project I set up a JavaScript for a Leaderboard list app, using webpack and ES6 features, notably modules. This is the basic structure of the application.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">HTML, CSS and JavaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Has basic structure to use.
+- You can view live edit in your localhost while editing.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+NPM installed
+Any code editors
+
+### Setup
+
+Clone this repository to your desired folder by.
+- cd my-folder
+- git clone [https://github.com/mershark/Leaderboard-website.git]
+- cd my-project
+
+
+### Install
+
+Installation is completed once the repository and cloned into your local machine.
+
+
+### Usage
+
+To run the project launch index.html in your browser
+
+### Run tests
+
+To run tests, run the following command:
+npx stylelint "**/*.{css,scss}
+
+### Deployment
+
+You can deploy this project by transferring the files in your local repository onto your server
+
```

Comment:
- [ ] Good job adding some information needed by users in order to use your application locally. However, I noticed that some of them are not correct and I think it would be better if you update them to make your documention be more professional and descriptive, such as how to: `access` the cloned directory, `install` dependencies, `run/use` your application,... To help you I suggest you **[refer this gist_fil](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)**e of how to fill all the above information

Reply:
Thank you @tresorsawasawa